public/app/features/alerting/unified/components/rules/RuleActionsButtons.tsx

```diff
@@ -123,6 +127,21 @@ export const RuleActionsButtons = ({ rule, rulesSource }: Props) => {
           />
         </Tooltip>
       );
+
+      moreActions.push(
+        <MenuItemPauseRule
+          rule={rule}
+          onPauseChange={async () => {
+            // Uses INSTANCES_DISPLAY_LIMIT + 1 here as exporting LIMIT_ALERTS from RuleList has the side effect
+            // of breaking some unrelated tests in Policy.test.tsx due to mocking approach
+            const limitAlerts = hasActiveFilters ? undefined : INSTANCES_DISPLAY_LIMIT + 1;
```

Comment:
Yeah, Policy.test started failing randomly if I imported `LIMIT_ALERTS` instead, I didn't unravel exactly why but it seemed like an issue with importing across multiple files + how the mocks were set up in there :shrug: